README.md

```diff
@@ -595,6 +596,95 @@ Output:
 }
 ```
 
+### WAV Wrap
+
+`WavWrap` tasks create a [WAV](https://en.wikipedia.org/wiki/WAV) file from an audio artifact, and apply data to specific chunks of the WAV wrapper.
+
+It accepts MPEG audio files, and any of the chunks supported by [prx-wavefile](https://github.com/PRX/prx-wavefile) including all Broadcast Wave Format chunks and `cart` chunks.
```

Comment:
Is there any good page that lists out all the possible fields for a cart chunk to link to? I looked, but couldn't find one.

Reply:
Sadly, the spec is contained in a doc that you have to pay to download from AES (audio engineering society).

I’m working on some better `cart` chunk docs - adding the details to `prx-wavefile` first, then we can link to that in porter

Reply:
Ah yeah, that's where I kept ending up. Ok sounds good!